content/hi/contributor-ladder/_index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: योगदानकर्ता सीढ़ी
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-pen-square'></i>
+---
+
+नमस्ते! 👋 CNCF क्लाउड नेटिव शब्दावली परियोजना में योगदान करने में आपकी रुचि के लिए धन्यवाद। इस समुदाय के सक्रिय सदस्य बनने के कई तरीके हैं जैसे नए शब्दों का योगदान देना, शब्दावली को अपनी मूल भाषा में स्थानीयकृत करने में मदद करना, या दूसरों को आरंभ करने में मदद करना। यह दस्तावेज़ परियोजना के भीतर विभिन्न योगदानकर्ता भूमिकाओं और उनके साथ आने वाली जिम्मेदारियों और विशेषाधिकारों की रूपरेखा तैयार करता है।
+
+## 1. योगदानकर्ता
+
+यह शब्दावली सभी के लिए है। कोई भी व्यक्ति केवल परियोजना में योगदान करके शब्दावली योगदानकर्ता बन सकता है। सभी योगदानकर्ताओं से [CNCF आचार संहिता] (https://github.com/cncf/foundation/blob/main/code-of-conduct.md) का पालन करने की अपेक्षा की जाती है।
+
+इस परियोजना में आप कई तरह से योगदान दे सकते हैं, जिनमें शामिल हैं:
+
+- **विषय योगदानकर्ता**: हर कोई जो मौजूदा शब्दों में सुधार करता है या नए योगदान देता है, 
+- **स्थानीयकरण योगदानकर्ता**: जो शब्दकोष का दूसरी भाषा में अनुवाद करने में मदद करते हैं,
+- **सहायक**: कोई भी व्यक्ति जो GitHub, Slack पर या जहाँ भी समुदाय के सदस्यों को समर्थन की आवश्यकता है, दूसरों की मदद करता है,
+- **राज-दूत**: जो कोई भी इस बात को फैलाने में मदद करता है, समुदाय को शिक्षित करता है कि कैसे और क्यों योगदान करना चाहिए। 
+
+योगदानकर्ताओं की कई भूमिकाएँ हो सकती हैं या वे केवल एक क्षेत्र पर ध्यान केंद्रित कर सकते हैं। 
+**ये सभी योगदान समान रूप से महत्वपूर्ण हैं** और एक संपन्न समुदाय को बढ़ावा देने में मदद करते हैं। कृपया विषय और स्थानीयकरण योगदान के लिए [योगदान कैसे करें](https://glossary.cncf.io/contribute/) और [शैली मार्गदर्शिका](https://glossary.cncf.io/style-guide/) देखें।
+
+## 2. अप्रूवर (Approvers) 
+
+अप्रूवर या अनुमोदनकर्ता, PR (पुल अनुरोध) पर प्रतिक्रिया प्रदान करते हैं और उन्हें अनुमोदित करते हैं। कोई भी सक्रिय योगदानकर्ता अप्रूवर बन सकता है (देखें "अप्रूवर बनना")। शब्दावली दो अप्रूवर के बीच अंतर करती है: (1) अंग्रेजी शब्दावली के लिए अप्रूवर और (2) स्थानीयकरण टीमों के लिए अप्रूवर 
+
+शब्दावली अनुमोदनकर्ताओं से अपेक्षा की जाती है कि वे:
```

Comment:
We should decide on one, अप्रूवर or अनुमोदनकर्ता or स्वीकृतकर्ता. Currently, we have all three being used on this page.
 
My suggestion would be to switch to अप्रूवर everywhere except for first line `अप्रूवर या अनुमोदनकर्ता, PR (पुल अनुरोध) पर प्रतिक्रिया प्रदान` as अनुमोदनकर्ता is not a very rarely used term.

Reply:
+1 to अप्रूवर

Reply:
Done.